src/foam/box/HTTPBox.js

```diff
@@ -225,70 +270,108 @@ let task = URLSession.shared.dataTask(with: request) { data, response, error in
 task.resume()
       */},
       javaCode: `
-// TODO: Go async and make request in a separate thread.
-java.net.HttpURLConnection conn;
-foam.box.Box replyBox = (foam.box.Box)msg.getAttributes().get("replyBox");
-
-try {
-  java.net.URL url = new java.net.URL(getUrl());
-  conn = (java.net.HttpURLConnection)url.openConnection();
-  conn.setDoOutput(true);
-  conn.setRequestMethod("POST");
-  conn.setRequestProperty("Accept", "application/json");
-  conn.setRequestProperty("Content-Type", "application/json");
+      // TODO: Go async and make request in a separate thread.
 
-  java.io.OutputStreamWriter output = new java.io.OutputStreamWriter(conn.getOutputStream(),
-                                                                     java.nio.charset.StandardCharsets.UTF_8);
+      java.net.HttpURLConnection conn;
```

Comment:
Java 11 comes with new http client api, not sure if we could use that instead.

Reply:
no, not changing this now.  We are going to replace anything with significant throughput with TCP client/server anyway.

Reply:
ok, thanks for the elaboration.